chapters/memory.txt

```diff
@@ -3096,18 +3096,175 @@ ifdef::VK_KHR_device_group[]
   * [[VUID-vkMapMemory-memory-00683]]
     pname:memory must: not have been allocated with multiple instances
 endif::VK_KHR_device_group[]
+ifdef::VK_EXT_map_memory_placed[]
+  * ename:VK_MEMORY_MAP_PLACED_BIT_EXT must: not be set in pname:flags
+endif::VK_EXT_map_memory_placed[]
 ****
 
 include::{generated}/validity/protos/vkMapMemory.txt[]
 --
 
-[open,refpage='VkMemoryMapFlags',desc='Reserved for future use',type='flags']
+ifdef::VK_EXT_map_memory_placed[]
+[open,refpage='VkMemoryMapFlagBits',desc='Bitmask specifying additional parameters of a memory map',type='enums']
+--
+Bits which can: be set in flink:vkMapMemory::pname:flags and
+slink:VkMemoryMapInfoEXT::pname:flags, specifying additional properties of
+a memory map, are:
+
+include::{generated}/api/enums/VkMemoryMapFlagBits.txt[]
+
+  * ename:VK_MEMORY_MAP_PLACED_BIT_EXT requests that the implementation
+    attempt to place the memory map at the virtual address specified by the
+    client via pname:ppData.
```

Comment:
```suggestion
    client via pname:ppData.
    Can: only be used in slink:VkMemoryMapInfoEXT::pname:flags.
```

Worth calling this out?

Reply:
We can, I guess.  There's a VU for that but we can call it out explicitly, I suppose.  IDK.

Reply:
I think a naive reader might get confused but probably not for long so eh, take it or leave it.